packages/web/src/components/tracks-table/TracksTable.tsx

```diff
@@ -249,14 +249,12 @@ export const TracksTable = ({
   const renderPlaysCell = useCallback(
     (cellInfo: TrackCell) => {
       const track = cellInfo.row.original
-      const { plays } = track
       const isOwner = track.owner_id === userId
-      // negative plays indicates the track is hidden
-      if (plays === -1) return '-'
       if (
-        track.is_stream_gated &&
-        isContentUSDCPurchaseGated(track.stream_conditions) &&
-        !isOwner
+        !isOwner &&
+        ((track.is_stream_gated &&
+          isContentUSDCPurchaseGated(track.stream_conditions)) ||
```

Comment:
what if I have purchased access?

Reply:
i think we still want to hide plays for gated content

Reply:
Ah ok